XMonad/Util/Loggers.hs

```diff
@@ -251,6 +254,70 @@ instance Default TitlesFormat where
     , urgentFormat    = wrap "!" "!" . xmobarRaw . shorten 30 . xmobarStrip
     }
 
```

Comment:
Could we perhaps make this a little bit more generic?  We're duplicating a lot of code right now